data/data/azure/main.tf

```
@@ -146,6 +146,7 @@ resource "azurerm_storage_account" "cluster" {
   location                 = var.azure_region
   account_tier             = "Standard"
   account_replication_type = "LRS"
+  min_tls_version          = (var.azure_environment == "public" || var.azure_environment == "usgovernment") ? "TLS1_2" : null
```

```suggestion
  min_tls_version          = contains(["public, "usgovernment"], var.azure_environment) ? "TLS1_2" : null
```
Or even move the `["public", "usgovernment"]` list to a local variable named something like `envirornments_with_tls1_1` to provide some hint from the name of the variable about why we are only choosing to use TLS 1.2 in those environments.